### PR TITLE
Remove src/python/graplinc; protobuf codegen creates correct `graplinc.` structure now.

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
None. Helping jesse with protobuf stuff.

### What changes does this PR make to Grapl? Why?
- `src/python/graplinc` was a snapshot of codegen, wildly out of date; I've removed it.
- pants is smart enough to automatically import `src/protos:graplinc` when it encounters a `from graplinc.` in Python code - _yes, really!_ - so we don't need explicit dependencies anymore (with reasonable exceptions like the repl)

### How were these changes tested?
CI and `make repl`
but the fact that previous users of `src/python/graplinc` can now use the new one, no issue - implies good things.

### CODE REVIEW

File: src/python/grapl_analyzerlib/grapl_analyzerlib/BUILD [hunk 1]
```diff
@@ -1,10 +1,5 @@
 python_library(
-    dependencies=[
-        # TODO: Ideally, this would be picked up by Pants' automatic
-        # dependency resolution, but something appears to be off with
-        # it.
-        "src/python/graplinc/graplinc/grapl/api"
-    ],
+    dependencies=[],
```
Comment (jgrillo-grapl) [inline src/python/grapl_analyzerlib/grapl_analyzerlib/BUILD]: I think this kwarg can go away completely

**Review by jgrillo-grapl — APPROVED** [review]
🥇

File: src/BUILD [hunk 2]
```diff
@@ -5,12 +5,11 @@
 grapl_python_distribution(
     name = "graplinc_grapl_api",
     dependencies = [
-        "src/proto/graplinc/grapl/api",
-        "src/python/graplinc/graplinc/grapl/api"
+        "src/proto:graplinc",
```
Comment (wimax-grapl) [inline src/BUILD:8]: yes, this technically includes more than we need to, and I could do a proto library that's just the api. Not convinced it's worth it.

File: src/python/grapl_analyzerlib/grapl_analyzerlib/BUILD (hunk 1 above)
Comment (wimax-grapl) [reply src/python/grapl_analyzerlib/grapl_analyzerlib/BUILD]: yep i noticed that :)

File: src/BUILD (hunk 2 above)
Comment (jgrillo-grapl) [reply src/BUILD:8]: Yes, I agree. I think all our protobuf stubs should be one "block", and ultimately they should be wrapped in a library. This library (`python-proto`, and its counterpart `rust-proto`) should be the sole point-of-contact for code which needs to serialize or deserialize protocol buffers. "Internal" code should not depend directly on the protobuf stubs.

This is one of the changes I'm attempting to introduce in https://github.com/grapl-security/grapl/pull/1121

**Pull request merged**